DESCRIPTION

```diff
@@ -44,7 +44,7 @@ Suggests:
     rlang,
     rmarkdown,
     rstudioapi (>= 0.2),
-    testthat (>= 3.0.0),
+    testthat (>= 3.1.5),
```

Comment:
`expect_no_warning()` was introduced in this version.

Reply:
I don't have any hard rule here, but something like "wait at least 6 months after CRAN release before depending on that version" seems reasonable, absent any pressing reason to have such a hot-off-the-press dependency.

So let's table this PR until the next release?

Reply:
Fine by me. 

`testthat 3.1.5` was released on 2022-10-08. So we should be hitting that 6-month mark pretty soon.

Reply:
good point! in fact at this point by the time we land a release we'll be there I reckon.